distributor-api-v1/use-cases/how-to-support-payment-cards-in-distributor-client-application.md

```diff
@@ -0,0 +1,19 @@
+# How to support payment cards in Distributor client application
+
+When creating a reservation it is possible to attach a payment card to the reservation.
+
+As part of creating your own custom Distributor client you may want to have a form within your client application where customers
+can enter their payment card data and send this data to Distributor API for further use. Payment card data can then
+be used by Mews in for charging the customer.
+
+To do that Distributor some API endpoints can support or require [CreditCardData](../operations.md#creditcarddata)
+in the request.
+
+### Where to get `CreditCardData` values
+
+* Confirm that **property supports PCI Proxy** by checking the field [PaymentCardStorageType](../operations.md#paymentcardstoragetype) in Distributor API response. Also read docs about [PaymentGateway](operations.md#payment-gateway) to see what other API data you could potentially use for your implementation.
```

Comment:
AFAIK all properties support PCI proxy. It's hardcoded on the backend.

Reply:
true for now, but this is basically a general check which is still present in the codebase, right?